minio_async/replicationconfig.py

```diff
@@ -107,7 +108,7 @@ def minutes(self):
         return self._minutes
 
     @classmethod
-    def from_xml(cls, element):
```

Comment:
What's the reasoning behind all of the `from_xml` to `fromxml` changes. The first one is adhering PEP more

Reply:
Oh once I noticed your code did't have `num_parallel_uploads`, and I thought It was because of version backward. So I refered to the latest minio and want to do some upgrade. They've changed some of the variable name, and changed some package. So I synced some of them. But I did't continue because it has too many things to change, and many of them were just name changing or formation changing, kind of meaningless. So this `from_xml` to `fromxml` is just a result of syncing.